### ISSUE TITLE
Consider switching to newer spatial thesauri

### ISSUE THREAD

**Issue by aappling-usgs** [USGS-R/meddle#62]
In a recent data review, @hcorson-dosch pointed out that FIPS codes have been withdrawn by NIST. See https://www.nist.gov/system/files/documents/itl/FIPSCodesReplacementChart2012.pdf and links below for details.

State/county codes (FIPS 5-2 and 6-4) have been [replaced](https://en.wikipedia.org/wiki/FIPS_county_code) by INCITS 38 and 31 (although the Census continues to use the same codes and calls them the FIPSeries rather than FIPStandard now, says Wikipedia? not clearly communicated [here](https://www.census.gov/library/reference/code-lists/ansi/ansi-codes-for-states.html), though). I tried to dig up those standards but they seem to be [behind a paywall](https://www.fgdc.gov/resources/download-geospatial-standards), which seems like a very strange way to do standards.

Country codes (FIPS 10-4) have been [replaced](https://en.wikipedia.org/wiki/FIPS_10-4) by ["Geopolitical Entities and Codes"](https://evs.nci.nih.gov/ftp1/GENC/) or are pretty similar to [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2). GEC, ISO3166, and STANAG codes are all listed [here](https://www.cia.gov/library/publications/resources/the-world-factbook/appendix/appendix-d.html#).

A third option for a new thesaurus to switch to is the USGS Common geographic areas thesaurus, which also includes hydrologic units: https://apps.usgs.gov/thesaurus/thesaurus-full.php?thcode=1

**Comment by jread-usgs** [comment]
Good catch, and I know you mentioned this on one of my earlier data release reviews that you did, but I didn't digest the full context like you have here. Seems worthy of an update/rework, and probably is a pretty confined task at least within the current code.
